lib/renderer.coffee

```diff
@@ -9,7 +9,7 @@ pandocHelper = null # Defer until used
 markdownIt = null # Defer until used
 {scopeForFenceName} = require './extension-helper'
 mathjaxHelper = require './mathjax-helper'
-pathWatcher = require 'pathwatcher-without-runas'
+pathWatcher = require atom.packages.resourcePath + '/node_modules/pathwatcher/lib/main'
```

Comment:
@leipert As this does seem to work it is probably better form, but from my test all that seemed to be nessesary was `require 'pathwatcher'` because Atom already search its own resource paths if the module is not installed locally for a package.